integration_tests/mockApis/hmppsAuth.ts

```diff
@@ -8,12 +8,14 @@ export interface UserToken {
 }
 
 function createToken(userToken: UserToken) {
+  // authorities in the user token will always be prefixed by ROLE_
```

Comment:
Basically mirror what happens in the app middleware when incoming roles are processed